app/app.go

```diff
@@ -191,16 +201,8 @@ func Run(ctx context.Context, conf Config) (err error) {
 }
 
 // wireP2P constructs the p2p tcp (libp2p) and udp (discv5) nodes and registers it with the life cycle manager.
-func wireP2P(ctx context.Context, life *lifecycle.Manager, conf Config, lock cluster.Lock,
+func wireP2P(ctx context.Context, life *lifecycle.Manager, conf Config, lock cluster.Lock, p2pKey *ecdsa.PrivateKey,
 ) (host.Host, *enode.LocalNode, error) {
-	p2pKey := conf.TestConfig.P2PKey
```

Comment:
why this was moved from here to Run function?

Reply:
previously this was only used in p2p wiring, now it is used in p2p and qbft (need to sign messages). So moved it one level up.